src/mail.sh

```diff
@@ -126,39 +126,39 @@ function validate_email()
 # the configuration
 #
 # @option:     The option being edited
-# @value:      The value to update the option to
 # @curr_scope: The scope being edited
 # @cmd_scope:  The scope being imposed on the commands
-# @set_option: The relevant index to access set_confs
 # @scope:      Used to go through all scopes
+# @values:     Array to store relevant values for each option
```

Comment:
I think this documentation does not match the function behavior; I cannot see value getting any parameter.

Reply:
`values` is passed to the `config_options` function that fills it with the values that are set in each scope and the value loaded in `options_values` for the given config